code/API_definitions/carrier_billing_payment.yaml

```diff
@@ -98,21 +88,19 @@ paths:
               schema:
                 $ref: '#/components/schemas/BodyServiceRequestError'

```

Comment:
Just curiosity, 

For 1-step payment error model is specific -> '#/components/schemas/BodyServiceRequestError' instead of using references to responses Error models (as in the rest of endpoints).

Not sure if preferred to keep as-is for any reason

Reply:
Good catch - I've simplified this.
For the jpg i'm stuck: I've the regenerated diagram but I'm not able to replace the png file.

Reply:
Let me think, to find a way. Will comment soon

Reply:
Let's do one thing, open another PR just with that png only. 
And we merge this before merging the other one.

Going to review now, in order to give OK